### PR TITLE
ack: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/ack.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# ack
+
+> Una herramienta de búsqueda como grep, optimizada para desarrolladores.
+> Ver también: `rg`, que es más rápido.
+> More information: <https://beyondgrep.com/documentation>.
+
+- Busca archivos que contengan una cadena o expresión regular en el directorio actual de forma recursiva:
+
+`ack "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:9]:
```suggestion
- Busca archivos que contengan una cadena o expresión regular en el directorio actual de forma recursiva:

`ack "{{patrón_de_búsqueda}}"`
```

Optional fix: placeholder can be translated.

File: pages.es/common/ack.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Buscar de un patrón sin distinción entre mayúsculas y minúsculas:
+
+`ack --ignore-case "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:13]:
```suggestion
- Buscar de un patrón sin distinción entre mayúsculas y minúsculas:

`ack --ignore-case "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Buscar líneas que coincidan con un patrón, imprimiendo s[o]lamente el texto coincidente y no el resto de la línea:
+
+`ack -o "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:17]:
```suggestion
- Buscar líneas que coincidan con un patrón, imprimiendo s[o]lamente el texto coincidente y no el resto de la línea:

`ack -o "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Limitar la búsqueda a archivos de un tipo específico:
+
+`ack --type={{ruby}} "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:21]:
```suggestion
- Limitar la búsqueda a archivos de un tipo específico:

`ack --type={{ruby}} "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- No buscar en archivos dado un tipo específico:
+
+`ack --type=no{{ruby}} "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:25]:
```suggestion
- No buscar en archivos dado un tipo específico:

`ack --type=no{{ruby}} "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- Contar el número total de coincidencias encontradas:
+
+`ack --count --no-filename "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:29]:
```suggestion
- Contar el número total de coincidencias encontradas:

`ack --count --no-filename "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Imprime sólo los nombres de los ficheros y el número de coincidencias de cada fichero:
+
+`ack --count --files-with-matches "{{search_pattern}}"`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:33]:
```suggestion
- Imprime sólo los nombres de los ficheros y el número de coincidencias de cada fichero:

`ack --count --files-with-matches "{{patrón_de_búsqueda}}"`
```

The same issue as above.

File: pages.es/common/ack.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- Lista de todos los valores que se pueden utilizar con `--type`.:
+
+`ack --help-types`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ack.md:37]:
```suggestion
- Lista de todos los valores que se pueden utilizar con `--type`:

`ack --help-types`
```

Optional fix: unexpected `.` can be removed.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🎄 🎅🏻

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/ack.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+> Una herramienta de búsqueda como grep, optimizada para desarrolladores.
+> Ver también: `rg`, que es más rápido.
+> More information: <https://beyondgrep.com/documentation>.
+
+- Busca archivos que contengan una cadena o expresión regular en el directorio actual de forma recursiva:
+
+`ack "{{patrón_de_búsqueda}}"`
+
+- Buscar de un patrón sin distinción entre mayúsculas y minúsculas:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:11]:
```suggestion
- Busca un patrón sin distinción entre mayúsculas y minúsculas:
```

File: pages.es/common/ack.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+`ack --ignore-case "{{patrón_de_búsqueda}}"`
+
+- Buscar líneas que coincidan con un patrón, imprimiendo s[o]lamente el texto coincidente y no el resto de la línea:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:15]:
```suggestion
- Busca líneas que coincidan con un patrón, imprimiendo s[o]lamente el texto coincidente y no el resto de la línea:
```

File: pages.es/common/ack.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+`ack -o "{{patrón_de_búsqueda}}"`
+
+- Limitar la búsqueda a archivos de un tipo específico:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:19]:
```suggestion
- Limita la búsqueda a archivos de un tipo específico:
```

File: pages.es/common/ack.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+`ack --type={{ruby}} "{{patrón_de_búsqueda}}"`
+
+- No buscar en archivos dado un tipo específico:
+
+`ack --type=no{{ruby}} "{{patrón_de_búsqueda}}"`
+
+- Contar el número total de coincidencias encontradas:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:27]:
```suggestion
- Cuenta el número total de coincidencias encontradas:
```

File: pages.es/common/ack.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+`ack --count --no-filename "{{patrón_de_búsqueda}}"`
+
+- Imprime sólo los nombres de los ficheros y el número de coincidencias de cada fichero:
+
+`ack --count --files-with-matches "{{patrón_de_búsqueda}}"`
+
+- Lista de todos los valores que se pueden utilizar con `--type`:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:35]:
```suggestion
- Lista todos los valores que se pueden utilizar con `--type`:
```

File: pages.es/common/ack.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+- No buscar en archivos dado un tipo específico:
+
+`ack --type=no{{ruby}} "{{patrón_de_búsqueda}}"`
+
+- Contar el número total de coincidencias encontradas:
+
+`ack --count --no-filename "{{patrón_de_búsqueda}}"`
+
+- Imprime sólo los nombres de los ficheros y el número de coincidencias de cada fichero:
```
Comment (navarroaxel) [inline pages.es/common/ack.md:31]:
```suggestion
- Imprime sólo los nombres de los archivos y el número de coincidencias de cada archivo:
```

**Pull request merged**